### ISSUE TITLE
Import failed, nullpointer instead of comprehensible error message

### ISSUE THREAD

**Issue by marijevdgeest** [molgenis/molgenis#4976]
#### Observed
```

[INFO] [talledLocalContainer] 10:25:07.644 [pool-9-thread-14] INFO  o.m.data.importer.ImportWriter - Rolling back changes.
[INFO] [talledLocalContainer] 10:25:11.462 [pool-9-thread-14] INFO  org.molgenis.data.importer.ImportJob - Import failed.
[INFO] [talledLocalContainer] java.lang.NullPointerException: null
[INFO] [talledLocalContainer] 	at org.molgenis.data.validation.ConstraintViolation.renumberRowIndex(ConstraintViolation.java:53) ~[molgenis-data-validation-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.validation.MolgenisValidationException.lambda$renumberViolationRowIndices$0(MolgenisValidationException.java:50) ~[molgenis-data-validation-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.validation.MolgenisValidationException$$Lambda$179/1006860325.accept(Unknown Source) ~[na:na]
[INFO] [talledLocalContainer] 	at java.util.Iterator.forEachRemaining(Iterator.java:116) ~[na:1.8.0_45]
[INFO] [talledLocalContainer] 	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[na:1.8.0_45]
[INFO] [talledLocalContainer] 	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:580) ~[na:1.8.0_45]
[INFO] [talledLocalContainer] 	at org.molgenis.data.validation.MolgenisValidationException.renumberViolationRowIndices(MolgenisValidationException.java:50) ~[molgenis-data-validation-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter.insertIntoRepo(ImportWriter.java:683) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter.update(ImportWriter.java:608) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter.importData(ImportWriter.java:228) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter.doImport(ImportWriter.java:122) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter$$FastClassBySpringCGLIB$$52fbfda4.invoke(<generated>) ~[spring-core-4.1.8.RELEASE.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204) ~[spring-core-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:717) ~[spring-aop-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157) ~[spring-aop-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.molgenis.security.core.runas.RunAsSystemProxy.invoke(RunAsSystemProxy.java:36) ~[molgenis-security-core-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99) ~[spring-tx-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:281) ~[spring-tx-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96) ~[spring-tx-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179) ~[spring-aop-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:653) ~[spring-aop-4.1.8.RELEASE.jar:4.1.8.RELEASE]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportWriter$$EnhancerBySpringCGLIB$$e959d833.doImport(<generated>) ~[spring-core-4.1.8.RELEASE.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.EmxImportService.doImport(EmxImportService.java:83) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.EmxImportService.doImport(EmxImportService.java:70) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at org.molgenis.data.importer.ImportJob.run(ImportJob.java:51) ~[molgenis-data-import-1.21.1.jar:1.21.1]
[INFO] [talledLocalContainer] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_45]
[INFO] [talledLocalContainer] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_45]
[INFO] [talledLocalContainer] 	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_45]
```

**Comment by fdlk** [comment]
ConstraintViolation sometimes doesn't contain a row number, renumbering fails then.

**Comment by dennishendriksen** [comment]
Fixed in 75cc61c213563d1dd6a376f5bdeadc571ddad032 for 2.0-RC
